apis/visibility/v1alpha1/types.go

```diff
@@ -27,6 +27,18 @@ import (
 type PendingWorkload struct {
 	metav1.TypeMeta   `json:",inline"`
 	metav1.ObjectMeta `json:"metadata,omitempty"`
+
+	// Priority indicates the workload's priority
+	Priority int32 `json:"priority"`
+
+	// LocalQueueName indicates the name of the LocalQueue the workload is submitted to
+	LocalQueueName string `json:"localQueueName"`
+
+	// PositionInClusterQueue indicates the workload's position in the ClusterQueue starting from 0
```

Comment:
```suggestion
	// PositionInClusterQueue indicates the workload's position in the ClusterQueue, starting from 1
```
I'm leaning to "Position" starting from 1 `1`:) , because this is a user-facing API (and "1" seems more natural to the user). I would be leaning towards 0 if this was "Index".

Reply:
Indeed we say in the KEP that offset starts from 0: `offset indicates position of the first fetched workload - default: 0`. In that case, I'm ok to leave it as is in this PR, but we could consider renaming in a follow up while the API is still alpha, based on user feedback.